scripts/generate_test_keys.sh

```diff
@@ -8,43 +8,43 @@
+openssl req -new -key testdata/ec-key1.pem -subj "/C=CA/ST=ON/O=Example Inc.:aries-framework-go/OU=aries-framework-go/CN=*.example.com" -reqexts SAN -config ../../../../test/fixtures/openssl/openssl.cnf -out testdata/ec-key1.csr
+openssl x509 -req -in testdata/ec-key1.csr -extensions SAN -extfile ../../../../test/fixtures/openssl/openssl.cnf -CA testdata/ec-cacert.pem -CAkey testdata/ec-cakey.pem -CAcreateserial -out testdata/ec-pubCert1.pem -days 365
 
 #extract pubkey 1
-openssl x509 -inform pem -in test/fixtures/keys/ec-pubCert1.pem -pubkey -noout > test/fixtures/keys/ec-pubKey1.pem
+openssl x509 -inform pem -in testdata/ec-pubCert1.pem -pubkey -noout > testdata/ec-pubKey1.pem
 
 #create creds 2
-openssl ecparam -name prime256v1 -genkey -noout -out test/fixtures/keys/ec-key2.pem
-openssl req -new -key test/fixtures/keys/ec-key2.pem -subj "/C=CA/ST=ON/O=Example Inc.:aries-framework-go/OU=aries-framework-go/CN=*.example.com" -reqexts SAN -config ./test/fixtures/openssl/openssl.cnf -out test/fixtures/keys/ec-key2.csr
-openssl x509 -req -in test/fixtures/keys/ec-key2.csr -extensions SAN -extfile ./test/fixtures/openssl/openssl.cnf -CA test/fixtures/keys/ec-cacert.pem -CAkey test/fixtures/keys/ec-cakey.pem -CAcreateserial -out test/fixtures/keys/ec-pubCert2.pem -days 365
+openssl ecparam -name prime256v1 -genkey -noout -out testdata/ec-key2.pem
+openssl req -new -key testdata/ec-key2.pem -subj "/C=CA/ST=ON/O=Example Inc.:aries-framework-go/OU=aries-framework-go/CN=*.example.com" -reqexts SAN -config ../../../../test/fixtures/openssl/openssl.cnf -out testdata/ec-key2.csr
+openssl x509 -req -in testdata/ec-key2.csr -extensions SAN -extfile ../../../../test/fixtures/openssl/openssl.cnf -CA testdata/ec-cacert.pem -CAkey testdata/ec-cakey.pem -CAcreateserial -out testdata/ec-pubCert2.pem -days 365
 
 #extract pubkey 2
-openssl x509 -inform pem -in test/fixtures/keys/ec-pubCert2.pem -pubkey -noout > test/fixtures/keys/ec-pubKey2.pem
+openssl x509 -inform pem -in testdata/ec-pubCert2.pem -pubkey -noout > testdata/ec-pubKey2.pem
 
 #create creds 3
-openssl ecparam -name prime256v1 -genkey -noout -out test/fixtures/keys/ec-key3.pem
-openssl req -new -key test/fixtures/keys/ec-key3.pem -subj "/C=CA/ST=ON/O=Example Inc.:aries-framework-go/OU=aries-framework-go/CN=*.example.com" -reqexts SAN -config ./test/fixtures/openssl/openssl.cnf -out test/fixtures/keys/ec-key3.csr
-openssl x509 -req -in test/fixtures/keys/ec-key3.csr -extensions SAN -extfile ./test/fixtures/openssl/openssl.cnf -CA test/fixtures/keys/ec-cacert.pem -CAkey test/fixtures/keys/ec-cakey.pem -CAcreateserial -out test/fixtures/keys/ec-pubCert3.pem -days 365
+openssl ecparam -name prime256v1 -genkey -noout -out testdata/ec-key3.pem
+openssl req -new -key testdata/ec-key3.pem -subj "/C=CA/ST=ON/O=Example Inc.:aries-framework-go/OU=aries-framework-go/CN=*.example.com" -reqexts SAN -config ../../../../test/fixtures/openssl/openssl.cnf -out testdata/ec-key3.csr
```

Comment:
why openssl.cnf in test/fixtures/openssl can we create dynamically

Reply:
(Or put it in testdata folder)

Reply:
deleted openssl.cnf file .. it is now temporarily generated in the script (and deleted at the end)